libbeat/reader/multiline/message_buffer.go

```diff
@@ -120,11 +123,15 @@ func (b *messageBuffer) addLine(m reader.Message) {
 // finalize writes the existing content into the returned message and resets all reader variables.
 func (b *messageBuffer) finalize() reader.Message {
 	if b.truncated > 0 {
-		b.message.AddFlagsWithKey("log.flags", "truncated")
+		if err := b.message.AddFlagsWithKey("log.flags", "truncated"); err != nil {
+			b.logger.Errorf("Failed to add truncated flag: %v", err)
+		}
```

Comment:
I'm not sure we would want this log message, if something goes wrong this can explode in cost for our users.